src/lib/ui/floatbar.ts

```diff
@@ -0,0 +1,74 @@
+import {LitElement, html, css, TemplateResult} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+
+@customElement('float-bar')
+export class FloatBar extends LitElement {
+    @property({type: Number}) float!: number;
+    @property({type: Number}) minFloat = 0;
+    @property({type: Number}) maxFloat = 1;
+
+    static styles = css`
+        .market-float-bar-container {
+            position: relative;
+            width: 100%;
+            height: 8px;
+            margin: 5px 0;
+        }
+
+        .market-float-bar-marker {
+            position: absolute;
+            background-color: #d9d9d9;
+            width: 3px;
+            top: -3px;
+            height: 14px;
+            border-radius: 4px;
+        }
+
+        .market-float-bar {
+            display: inline-block;
+            vertical-align: top;
+            height: 100%;
+            opacity: 0.8;
+        }
+
+        .market-float-bar:not([style*='background-color: transparent']):first-of-type {
```

Comment:
I don't believe the `:not(...)` selector is needed here since we don't have an instance of setting an element to a transparent background color?

AFAIK, if a certain wear range isn't applicable, it's getting a 0% width below.

Reply:
Yes, you are absolutely right. I played around with making it transparent first, but changing the width seemed more consistent.